contrib/mac-build

```diff
@@ -6,15 +6,16 @@
 #
 
 macports_prereq="autoconf automake boost gmp gtime libtool readline"
+homebrew_prereq="autoconf automake boost gmp gnu-time libtool readline"
```

Comment:
Unfortunately, Homebrew and MacPorts have different names for GNU Time, so the list needs to be duplicated.